packages/scandipwa/src/component/MyAccountCreateAccount/MyAccountCreateAccount.component.js

```diff
@@ -159,10 +162,23 @@ export class MyAccountCreateAccount extends PureComponent {
                           inputType: VALIDATION_INPUT_TYPE.password,
                           match: (value) => {
                               const email = document.getElementById('email');
-                              return value && email.value !== value;
-                          },
-                          customErrorMessages: {
-                              onMatchFail: __('Passwords can\'t be the same as email!')
+
+                              if (value && email.value === value) {
+                                  return __('Passwords can\'t be the same as email!');
+                              }
+
+                              const counter = !!(value.match(/\d+/))
```

Comment:
If you want to just check that string matches regex then you can use `/\d+/.test(value)` which will return boolean right away

Reply:
Also, if you work with numbers here you should probably use numbers as it will be much easier to understand what this part of the code means